src/card.css

```diff
@@ -0,0 +1,83 @@
+* {
+  box-sizing: border-box;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+
+  font-family: "Roboto", Helvetica, sans-serif;
+  color: #060b35;
+  font-weight: 400;
+  font-size: 10px;
+  border-radius: 5px;
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
+  margin: 32px 19px 40px;
+}
+
+.card__information,
+.card__price,
+.card__rating,
+.card__button {
+  margin: 0 16px 16px;
+}
+
+.card__rating,
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.card__rating {
+  margin-bottom: 24px;
+}
+
+.card__information h2 {
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+}
+
+.card__information p {
```

Comment:
better 2 use `<p>` with .card__information class `<p class="card__information">` or something, like in div.card__price imho not sure, so u may not do it